File: Source/ecs/Source/Library/philipsecommercesdk/src/main/java/com/philips/cdp/di/ecs/model/summary/Brand.java
package com.philips.cdp.di.ecs.model.summary;

/**
 * Description :
 * Project : PRX Common Component.
 * Created by naveen@philips.com on 02-Nov-15.
 */

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;


public class Brand  implements Serializable {

    @SerializedName("brandLogo")
    @Expose
    private String brandLogo;

    /**
     * No args constructor for use in serialization
     *
     */
    public Brand() {
    }

    /**
     *
     * @param brandLogo
     */
    public Brand(String brandLogo) {
        this.brandLogo = brandLogo;
    }

    /**
     *
     * @return
     * The brandLogo
     */
    public String getBrandLogo() {
        return brandLogo;
    }

    /**
     *
     * @param brandLogo
     * The brandLogo
     */
    public void setBrandLogo(String brandLogo) {
        this.brandLogo = brandLogo;
    }

}

File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/summary/Brand.kt
package com.philips.cdp.prxclient.datamodels.summary

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Description :
 * Project : PRX Common Component.
 */
@Parcelize
data class Brand(val partnerBrandType: String?, val partnerLogo: String?, val brandLogo: String?) : Parcelable